### PR TITLE
Blessed First Edit

### PR DESCRIPTION
**Pull request by blestab**

### CODE REVIEW

**Comment by blestab** [comment]
Here is my pull request

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # first
 My First GitHub Project
```
Comment (gman3g) [inline README.md:2]: I do not want this line back. I think we should remove this for the final version.

**Review by gman3g — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (blestab) [inline README.md]: Agreed

**Review by blestab — COMMENTED** [review]
Agreed

**Review by gman3g — APPROVED** [review]

**Pull request merged**